pilot/utils/llm_connection.py

```diff
@@ -76,9 +77,9 @@ def test_api_access(project) -> bool:
     ]
 
     endpoint = os.getenv('ENDPOINT')
-    model = os.getenv('MODEL_NAME', 'gpt-4')
+    model = os.getenv('DEFAULT_MODEL_NAME', DEFAULT_MODEL_NAME)
```

Comment:
This will get a bit more complicated since we merged Anthropic support and also have a notion of provider and model name split (eg `anthropic` / `claude-3-...`). This is not actually used in `create_chat_gpt_completion` because currently it looks up the model directly, but your change will make that part of the code work slightly different. I'll dive deeper into the side effect when you resolve the conflicts and I can test more.